lib/archethic_web/live/chains/node_shared_secrets_live.ex

```diff
@@ -0,0 +1,221 @@
+defmodule ArchethicWeb.NodeSharedSecretsChainLive do
+  @moduledoc false
+
+  use ArchethicWeb, :live_view
+
+  alias Archethic.{
+    TransactionChain,
+    PubSub,
+    SharedSecrets
+  }
+
+  alias ArchethicWeb.{ExplorerView}
+  alias Phoenix.{LiveView, View}
+
+  @display_limit 10
+  @txn_type :node_shared_secrets
+
+  @spec mount(_parameters :: map(), _session :: map(), socket :: LiveView.Socket.t()) ::
+          {:ok, LiveView.Socket.t()}
+  def mount(_parameters, _session, socket) do
+    if connected?(socket) do
+      PubSub.register_to_new_transaction_by_type(@txn_type)
+    end
+
+    tx_count = TransactionChain.count_transactions_by_type(@txn_type)
+
+    socket =
+      case @txn_type
+           |> SharedSecrets.genesis_address() do
+        nil ->
+          socket
+          |> assign(:tx_count, 0)
+          |> assign(:nb_pages, 0)
+          |> assign(:nb_authorized_nodes, 0)
+          |> assign(:current_page, 1)
+          |> assign(:transactions, [])
+
+        address when is_binary(address) ->
+          nb_authorized_nodes =
+            address
+            |> TransactionChain.get_last_transaction()
```

Comment:
Here you can use the function `nb_of_authorized_keys`

Reply:
...